lib/new_relic/agent/configuration/default_source.rb

```diff
@@ -1715,6 +1716,31 @@ def self.enforce_fallback(allowed_values: nil, fallback: nil)
           :transform => DefaultSource.method(:convert_to_regexp_list),
           :description => 'Define transactions you want the agent to ignore, by specifying a list of patterns matching the URI you want to ignore. For more detail, see [the docs on ignoring specific transactions](/docs/agents/ruby-agent/api-guides/ignoring-specific-transactions/#config-ignoring).'
         },
+        # Sidekiq
+        :'sidekiq.args.include' => {
+          default: NewRelic::EMPTY_ARRAY,
+          public: true,
+          type: Array,
+          allowed_from_server: false,
+          description: 'An array of strings that will collectively serve as an allowlist for filtering which Sidekiq ' +
+            'job arguments get reported to New Relic. The capturing of any Sidekiq arguments requires that ' +
+            "'job.sidekiq.args.*' be added to the separate :'attributes.include' configuration option. Each " +
+            'string in this array will be turned into a regular expression via `Regexp.new` to permit advanced ' +
+            'matching. For job argument hashes, if either a key or value matches the pair will be included. All ' +
+            'matching job argument array elements and job argument scalars will be included.'
```

Comment:
Out of curiosity, why did you choose this approach over a heredoc?

Reply:
2 reasons:

- Originally I just used one single, huge long line to match the style of the existing entries. But I just so happened to go over the previous record for maximum line length that we have set in our RuboCop todo file, so I then changed it to not cross 120 columns anywhere.
- I have still been operating as though we need to support Ruby 2.2, which doesn't support squiggly heredocs. But now that v9.x of the agent only supports Ruby 2.4+, they're a totally valid option that I should leverage.

Reply:
Heredocs used with [fa500e4](https://github.com/newrelic/newrelic-ruby-agent/pull/2177/commits/fa500e436fced67a32e7fea7d06dcbd739a97b0e)